Assets/Resources/Microgames/KaguyaMemory/Scripts/KaguyaMemory_ItemScript.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class KaguyaMemory_ItemScript : MonoBehaviour {
+
+    [SerializeField]
+    private AudioClip correctSound;
+
+    [SerializeField]
+    private AudioClip wrongSound;
+
+    public GameObject rngMaster;
+    private Vector3 startingPosition;
+    private int timer1 = 1;
+    public bool isMoving = false;
+    public bool isCorrect = false;
+    public GameObject correctIndicator;
+    public GameObject wrongIndicator;
+    public bool isClicked = false;
```

Comment:
It would be nice if your variables were ordered: public then private. That's what the C# guidelines recommend and it makes them a little easier to read.